dev-infrastructure/scripts/istio.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+
+set -euxo pipefail
+
+echo "********** ISTIO Upgrade Started **************"
+# Get the current istio and check if it match target version
+export CURRENTVERSION=$(kubectl get pods -n aks-istio-system | tail -n +2 | tail -n 1 | cut -d '-' -f 2,3,4)
```

Comment:
General statement - never use string manipulation on structured data. e.g. here, please `-o json` and filter with `jq` or use `-o jsonpath`. Select what objects you want with `--selector`.